doc/doxygen/Doxyfile

```diff
@@ -851,7 +851,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = ../../include/pagmo/external ../../include/pagmo/detail
+EXCLUDE                = ../../include/pagmo/external ../../include/pagmo/detail/cec2013_data.hpp
```

Comment:
Do we want to exclude only ``cec2013_data.hpp`` here from doxygen, and not the whole ``detail/`` subdir? Am I reading this correctly?